src/librustc_unicode/char.rs

```diff
@@ -551,6 +551,31 @@ impl char {
         }
     }
 
+    /// Returns true if this `char` is an a letter code point, and false if not.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// assert!('a'.is_letter());
+    /// assert!('京'.is_letter());
+    ///
+    /// let c = '💝';
+    /// // love is many things, but it is not a letter
+    /// assert!(!c.is_letter());
+    /// assert!(!'5'.is_letter());
+    /// ```
+    #[stable(feature = "rust1", since = "1.15.0")]
```

Comment:
I see. Marked unstable and gave it its own name. There's no tracking issue for marking this stable, so set it to "0", as I saw with other features.